src/draw/renesas/dave2d/lv_draw_dave2d_utils.c

```diff
@@ -100,6 +100,7 @@ d2_u32 lv_draw_dave2d_lv_colour_fmt_to_d2_fmt(lv_color_format_t colour_format)
 
     switch(colour_format) {
         case(8):
```

Comment:
[8 means  `LV_COLOR_FORMAT_I2 `](https://github.com/lvgl/lvgl/blob/master/src/misc/lv_color.h#L123). I think it's incorrect.

Reply:
I have updated the pull request, removing the case(8)